extension/_locales/es/messages.json

```diff
@@ -384,7 +384,7 @@
     "description": "List Type"
   },
   "domainPlaceholderText": {
-    "message": "ejemplo.com, subdominio.ejemplo.com, *.ejemplo.com",
+    "message": "ejemplo.com, subdominio.ejemplo.com, *.ejemplo.com, /(^|.)ejemplo\\.com/",
```

Comment:
I don't think this string should be allowed to be translated. Maybe only `subdomain` part.

https://en.wikipedia.org/wiki/Example.com

Reply:
Good point.  I'll update it in a future version to be static.